src/IssueViz.Security/IssueViz.Security.csproj

```diff
@@ -32,6 +32,17 @@
 
   </PropertyGroup>
 
+  <ItemGroup Label="Taint Xaml Sources">
+    <None Remove="Taint\TaintList\sonarlint.png" />
+    <Resource Include="Taint\TaintList\sonarlint.png" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Content Include="Commands\Resources\sonarlint.png">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </Content>
+  </ItemGroup>
```

Comment:
You've reverted part of the change to `Commands\Resources\sonarlint.png` in the project file but not this bit.
This file isn't referenced by any of your code changes, is it? Is it needed?